src/components/export/export.js

```diff
@@ -99,14 +92,13 @@ export default class Export extends Component {
 
     updateFT() {
 
+        if (this.errors.token.isBad) return;
```

Comment:
I think it would make sense to return after setting `this.errors.noToken.isBad` and `this.errors.notWhitelisted.isBad` to `false`.